drivers/soundwire/bus.c

```diff
@@ -1264,9 +1272,19 @@ static int sdw_initialize_slave(struct sdw_slave *slave)
 	if (slave->bus->prop.quirks & SDW_MASTER_QUIRKS_CLEAR_INITIAL_CLASH) {
 		/* Clear bus clash interrupt before enabling interrupt mask */
 		status = sdw_read_no_pm(slave, SDW_SCP_INT1);
```

Comment:
there's one more unchecked call to `sdw_read_no_pm()` below at line 1275, and a `sdw_write_no_pm()` after it.

Reply:
nice catch, thanks for the review @lyakh

Reply:
fixed now